images/installer/nested-libvirt/mock-nss.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# mock passwd and group files
+(
+  exec 2>/dev/null
+  username="${NSS_USERNAME:-$(id -un)}"
+  uid="${NSS_UID:-$(id -u)}"
+
+  groupname="${NSS_GROUPNAME:-$(id -gn)}"
+  gid="${NSS_GID:-$(id -g)}"
+
+  echo "${username}:x:${uid}:${uid}:gecos:${HOME}:/bin/bash" > "${NSS_WRAPPER_PASSWD}"
+  echo "${groupname}:x:${gid}:" > "${NSS_WRAPPER_GROUP}"
```

Comment:
Can you document `NSS_WRAPPER_PASSWD` and `NSS_WRAPPER_GROUP` in the README for this directory?  They're mentioned [here][1], but your use here is outside of the library, and we don't have defaults, so the caller will need to set them.  Does your `podman run ...` example work without them?

[1]: https://cwrap.org/nss_wrapper.html

Reply:
no it does not, added/fixed.